cpp/ps0007/ps0007_solv.cpp

```diff
@@ -179,3 +179,4 @@ float area_calc (char shape , float dimensions[])
         }
        return 0;
    }
+// great work 😎 8.5/10 ⭐

```

Comment:
- [x] program is in the correct path
- [ ] no syntax errors while compilation & no bugs or warnings while execution
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code
- [x] used array & function concepts for the program
- [ ] good design for function using DRY concept
- [ ] main returns non-zero/zero number according to the case
+0.5 bonus mark for your effort ⭐

you can check the [problem solution](https://onlinegdb.com/wybZ67yJX) for more